### ISSUE TITLE
Libboost 1.58 deprecated

### ISSUE THREAD

**Issue by ghost** [wavidev-the-man/wavi#8]
The wallet binaries are compiled using boost 1.58. boost 1.58 is quite hard to find in modern distribution. Maybe static link the binaries?

**Comment by tconrado** [comment]
yes, please! I would love to have a standalone package, in which no libraries has to be installed...
it is our third client that we turn donw on our service, because we have the policy of only supporting coins that run on Ubuntu 18 withour ANY additional lib installation on the system... 

please guys publish that, we know that it is easy for the developer team!
